_haikus/cyberpunk.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Cyberpunk
+author: Jemaz
+---
+
+SSH, jacking in, handshake <br>
+Cyberpunk streaming neon lights <br>
```

Comment:
This line also has one syllable too many, you can amend with:

```
Cyberpunk streaming neon <br>
```

Reply:
Sorry for the (extremely) slow reply. 
I will take your feedback and reopen a new pull request.
Thanks.